### PR TITLE
Gather and return real statistics in Stats()

### PR DESCRIPTION
**Pull request by adunham-stripe**
r? @Random-Liu 

You can see the stats that gvisor itself returns here:
https://github.com/google/gvisor/blob/master/runsc/boot/events.go#L61-L81

### CODE REVIEW

**Comment by adunham-stripe** [comment]
@Random-Liu - Ping! Anything I can do to help get this merged?

File: pkg/v2/service.go [hunk 1]
```diff
@@ -565,8 +565,37 @@ func (s *service) Shutdown(ctx context.Context, r *taskAPI.ShutdownRequest) (*pt
...
+	}
+	ns, err := namespaces.NamespaceRequired(ctx)
+	if err != nil {
+		return nil, err
+	}
+	runtime, err := s.readRuntime(path)
+	if err != nil {
+		return nil, err
+	}
+	rs := proc.NewRunsc(proc.RunscRoot, path, ns, runtime, nil)
```
Comment (fvoznika) [inline pkg/v2/service.go]: At this point, a runsc process should already exist:
```
	p, err := s.getProcess(r.ExecID)
	if err != nil {
		return nil, err
	}
```

File: pkg/v2/service.go [hunk 2]
```diff
@@ -565,8 +565,37 @@ func (s *service) Shutdown(ctx context.Context, r *taskAPI.ShutdownRequest) (*pt
...
+	stats, err := rs.Stats(ctx, s.id)
+	if err != nil {
+		return nil, err
+	}
+
+	// gvisor currently (as of 2020-02-28) only returns the total memory
+	// usage and current PID value. Copy those into the containerd cgroup
+	// statistics structure so that we're returning the same type as runc.
+	data, err := typeurl.MarshalAny(&cgroups.Metrics{
```
Comment (fvoznika) [inline pkg/v2/service.go]: Can you fill up all fields in cgroups.Metrics? This way, when new data is returned from runsc, they are automatically propagated from here.

Comment (adunham-stripe) [reply pkg/v2/service.go]: Done!

File: pkg/v2/service.go (hunk 1 above)
Comment (adunham-stripe) [reply pkg/v2/service.go]: Yes, the process should already exist, but the `Stats` method is only exposed on the container, not on [`rproc.Process`](https://pkg.go.dev/github.com/containerd/containerd/runtime/proc?tab=doc#Process).  Unless I'm misinterpreting what you're meaning by this comment?

Also, as a side note, gvisor appears to [ignore](https://github.com/google/gvisor/blob/277a0d5a1fbe8272d4729c01ee4c6e374d047ebc/runsc/boot/events.go#L62) the input anyway and return container-wide metrics in all cases?

Comment (fvoznika) [reply pkg/v2/service.go]: Never mind. I thought for a second that getProcess would return `*runsc.Runsc`.

Yes, gVisor doesn't track per-container stats at the moment (https://github.com/google/gvisor/issues/172).

**Comment by adunham-stripe** [comment]
@fvoznika - Thank you! Much appreciated 😁

**Pull request merged**